pytest_socket.py

```diff
@@ -170,16 +170,52 @@ def host_from_connect_args(args):
         return host_from_address(address)
 
 
+def is_ipaddress(address: str):
```

Comment:
If we're gonna type, let's type!
```suggestion
def is_ipaddress(address: str) -> bool:
```